### ISSUE TITLE
Cannot find module 'monocdk/lib/aws-logs/lib/log-group' or its corresponding type declarations

### ISSUE THREAD

**Issue by multimeric** [aws/amazon-genomics-cli#194]
**Describe the Bug**

When I run `agc account activate`, I get:

```
2021-11-29T14:44:14+11:00 ✘  TSError: ⨯ Unable to compile TypeScript:
2021-11-29T14:44:14+11:00 ✘  ../../lib/util/index.ts(12,27): error TS2307: Cannot find module 'monocdk/lib/aws-logs/lib/log-group' or its corresponding type declarations.
```
(see full log below)

**Steps to Reproduce**

* I installed an old version of the AGC, possibly 1.0
* I then re-ran the installer for the AGC 1.1.1
* Finally, I ran `agc account activate` as above

**Relevant Logs**

```
2021-11-29T14:44:00+11:00 𝒊  Activating AGC with bucket '' and VPC ''
Activating account... [_______________________________________________________________________________________________________________________________________________________________________________________]9.0s2021-11-29T14:44:14+11:00 ✘  /home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:750
2021-11-29T14:44:14+11:00 ✘      return new TSError(diagnosticText, diagnosticCodes);
2021-11-29T14:44:14+11:00 ✘             ^
2021-11-29T14:44:14+11:00 ✘  TSError: ⨯ Unable to compile TypeScript:
2021-11-29T14:44:14+11:00 ✘  ../../lib/util/index.ts(12,27): error TS2307: Cannot find module 'monocdk/lib/aws-logs/lib/log-group' or its corresponding type declarations.
2021-11-29T14:44:14+11:00 ✘  
2021-11-29T14:44:14+11:00 ✘      at createTSError (/home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:750:12)
2021-11-29T14:44:14+11:00 ✘      at reportTSError (/home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:754:19)
2021-11-29T14:44:14+11:00 ✘      at getOutput (/home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:941:36)
2021-11-29T14:44:14+11:00 ✘      at Object.compile (/home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:1243:30)
2021-11-29T14:44:14+11:00 ✘      at Module.m._compile (/home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:1370:30)
2021-11-29T14:44:14+11:00 ✘      at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
2021-11-29T14:44:14+11:00 ✘      at Object.require.extensions.<computed> [as .ts] (/home/migwell/.agc/cdk/node_modules/ts-node/src/index.ts:1374:12)
2021-11-29T14:44:14+11:00 ✘      at Module.load (node:internal/modules/cjs/loader:981:32)
2021-11-29T14:44:14+11:00 ✘      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
2021-11-29T14:44:14+11:00 ✘      at Module.require (node:internal/modules/cjs/loader:1005:19) {
2021-11-29T14:44:14+11:00 ✘    diagnosticText: "../../lib/util/index.ts(12,27): error TS2307: Cannot find module 'monocdk/lib/aws-logs/lib/log-group' or its corresponding type declarations.\n",
2021-11-29T14:44:14+11:00 ✘    diagnosticCodes: [ 2307 ]
2021-11-29T14:44:14+11:00 ✘  }
2021-11-29T14:44:14+11:00 ✘  Subprocess exited with error 1
2021-11-29T14:44:14+11:00 ✘   error="exit status 1"
Error: an error occurred invoking 'account activate'
with variables: {bucketName: vpcId:}
caused by: exit status 1
```
Operating System:
AGC Version: 1.1.1
Was AGC setup with a custom bucket: No
Was AGC setup with a custom VPC: No

**Comment by AndreyDovydenko** [comment]
Hello multimeric!

This is resolved in https://github.com/aws/amazon-genomics-cli/pull/190 please feel free to pull from the sources, we would release a patch version including it.

**Comment by multimeric** [comment]
Thanks, I missed that PR. Will I have to rebuild from source, or can I just pull from main in a certain repository? (`~/.agc` doesn't seem to be a git repo, though).

**Comment by nbraid** [comment]
Hey @multimeric, this should be fixed in v1.1.2. Please try downloading that patch build and reopen this issue if that doesn't resolve the problem. Thanks!

https://github.com/aws/amazon-genomics-cli/releases/tag/v1.1.2
